core/src/main/java/org/elasticsearch/common/logging/LogConfigurator.java

```diff
@@ -51,9 +55,29 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.StreamSupport;
 
 public class LogConfigurator {
 
+    private static final AtomicBoolean error = new AtomicBoolean();
```

Comment:
The most likely error that is logged here is that we touched a logger before it's configured, but any error message from the status logger is going to trip this check so I'd rather keep the name general. I'm happy to add a comment that explains this though.